doc/sphinx/src/getting-started.rst

```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
 
   git clone --recursive git@github.com:lanl/singularity-eos.git
   cd singularity-eos
-  mkdir bin
-  cd bin
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
+  mkdir -p build && cd build
+  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON ..
   make -j
   make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```

Comment:
```suggestion
  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON 
  cmake --build builddir --parallel
  cmake --install builddir # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```
Make it more modern CMake.

Reply:
Do you object to using `build` instead of `builddir`?

Reply:
No objection, my experience is just that user always confuse ``--build`` and `build`` in `cmake --build build`, one is an argument, one is a directory.